lib/collection/src/operations/types.rs

```diff
@@ -723,8 +722,8 @@ impl From<JsonError> for CollectionError {
     }
 }
 
-impl From<io::Error> for CollectionError {
-    fn from(err: io::Error) -> Self {
+impl From<futures::io::Error> for CollectionError {
```

Comment:
Example of conflict found for `io` import.
Decided to go with full name here.

Reply:
We have other places with conflicts like this (not related to `io`). So I don't think this is too much of a worry.